### PR TITLE
Support Rails 5.1

### PR DESCRIPTION
**Pull request by maschwenk**
Rails 5.1 release has been officially cut http://weblog.rubyonrails.org/2017/4/27/Rails-5-1-final/

### CODE REVIEW

**Comment by maschwenk** [comment]
Everyone has a different policy about bumping version/changelog directly in a PR. Let me know if I should revert that part.

File: activerecord-session_store.gemspec [hunk 1]
```diff
@@ -6,7 +6,7 @@ Gem::Specification.new do |s|
   s.version     = ActiveRecord::SessionStore::VERSION
   s.summary     = 'An Action Dispatch session store backed by an Active Record class.'
 
-  s.required_ruby_version = '>= 1.9.3'
```
Comment (rafaelfranca) [inline activerecord-session_store.gemspec]: We still need to support 1.9.3

File: lib/active_record/session_store/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module ActiveRecord
   module SessionStore
-    VERSION = '1.0.0'
+    VERSION = '1.1.0'
```
Comment (rafaelfranca) [inline lib/active_record/session_store/version.rb]: Please don't bump the version here

File: activerecord-session_store.gemspec (hunk 1 above)
Comment (maschwenk) [inline activerecord-session_store.gemspec]: Ahh, looking at the failures in CI now I understand what needs to be done. Will fix.

**Comment by maschwenk** [comment]
@rafaelfranca I updated the appraisals to make sure Nokogiri is locked to 1.6 for Rails < 5 to restore Ruby 1.9.x support

**Comment by chrisarcand** [comment]
@rafaelfranca Can a release be made with these changes, please?

**Pull request merged**